specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -149,36 +146,22 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			);
 		} );
 
-		step( 'Can see the post category in preview', async function() {
-			const gPreviewComponent = await GutenbergPostPreviewComponent.Expect( driver );
-			let categoryDisplayed = await gPreviewComponent.categoryDisplayed();
+		step( 'Can see the post tag in preview', async function() {
```

Comment:
@alisterscott I disabled this step in https://github.com/Automattic/wp-e2e-tests/pull/1712 Test started to fail on this step after @Copons merged https://github.com/Automattic/wp-calypso/pull/29321 and https://github.com/Automattic/wp-e2e-tests/pull/1710

Should we check category in preview? I see it's removed now.